components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -52,29 +52,30 @@ interface Props {
 export const PrebuildDetailPage: FC = () => {
     const { prebuildId } = useParams<Props>();
 
-    const { data: prebuild, isLoading: isInfoLoading, error, refetch } = usePrebuildQuery(prebuildId);
+    const { data: initialPrebuild, isLoading: isInfoLoading, error, refetch } = usePrebuildQuery(prebuildId);
+    const [currentPrebuild, setCurrentPrebuild] = useState<Prebuild | undefined>();
+    const prebuild = currentPrebuild ?? initialPrebuild;
```

Comment:
This I think makes the page so much simpler. We used to use `initialPrebuild` somewhere and `currentPrebuild` elsewhere, but there was not a good reason why. Unifying the "watched" and "initially fetched" prebuild into `prebuild` should make everything here more readable.